### PR TITLE
[#368] Initial efforts to rename Packet to Equinix Metal in CNF Testbed

### PR DESCRIPTION
**Pull request by michaelspedersen**
Part of #368 

* Focus is on documentation that refers to Packet.
* For API endpoints, Packet is still used as it is still available.

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ If you want to deploy the CNF-Testbed in the same way that our CI job does, you
 can do that by using the Makefile and following the steps below.
 
 ## Pre-requisites:
-Set your Packet.net project name, id & api tokens.
+Set your [Equinix Metal] project name, id & api tokens. These can be found through the [Console].
```
Comment (lixuna) [inline tools/README.md:7]: Did we want to add a URL for the [Console]?

File: comparison/ansible/packet_layer2.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 - hosts: localhost
   tasks:
-    - name: Configure Packet switch for quad port intel machines
+    - name: Configure Eqinix Metal switch for quad port intel machines
```
Comment (lixuna) [inline comparison/ansible/packet_layer2.yml:4]: "Eqinix Metal" should be "Equinix Metal"

File: tools/README.md (hunk 1 above)
Comment (lixuna) [reply tools/README.md:7]: Please disregard, I see below: 
- [Console]: http://console.equinix.com/ "Console"

LGTM

**Comment by lixuna** [comment]
Thank you for updating these 53 files, @michaelspedersen 

I marked the files as viewed and only found one issue:
- typo `Eqinix Metal` should be `Equinix Metal` at https://github.com/cncf/cnf-testbed/pull/369/files?short_path=a7af234#diff-f3b1412ddcffe51d93b0b4d5352fdb7ba797f773f50ca690bf84d0d744e0fd13R4

File: comparison/ansible/packet_layer2.yml (hunk 2 above)
Comment (taylor) [reply comparison/ansible/packet_layer2.yml:4]:
```suggestion
    - name: Configure Equinix Metal switch for quad port intel machines
```

**Review by taylor — CHANGES_REQUESTED** [review]

**Pull request merged**